crates/sui-faucet/src/main.rs

```diff
@@ -52,6 +52,11 @@ struct FaucetConfig {
 
     #[clap(long, default_value_t = 10)]
     max_request_per_second: u64,
+
+    // Setting timeout to be higher than fullnode maximum server-side timeout is a no-op.
+    // E.g. execute_transaction() has a 30s server-side timeout.
+    #[clap(long, default_value_t = 60)]
+    timeout_secs: u64,
```

Comment:
nit:People might confuse this timeout with the request timeout in Axum

```suggestion
    wallet_client_timeout_secs: u64,
```